ompi/mca/hook/demo/hook_demo_fns.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2016      IBM Corporation.  All rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#include "ompi_config.h"
+
+#include "hook_demo.h"
```

Comment:
Should probably also include `opal_output.h` in here, since you call `opal_output()`.